kumascript/macros/APIListAlpha.ejs

```diff
@@ -81,17 +81,17 @@ for (var i = 0; i < numLetters; i++) {
 
             // add badges if needed
 
-            if (containsTag(tags, "Non-standard") || containsTag(tags, "Non standard")) {
+            if (pages[p].status.includes("non-standard") || containsTag(tags, "Non-standard") || containsTag(tags, "Non standard")) {
```

Comment:
FYI, this is just used in the Web/API home page for the "Interfaces" section

Reply:
Nit: Let's replace lines 67 to 69 as follows:
```patch
-            var tags = pages[p].tags;
-            var url = pages[p].url;
-            var title = pages[p].title;
+            const { status, tags, title, url } = pages[p];
```

Then we can use `status` here and below.

Reply:
Yes, looks better. Done.